### PR TITLE
Bloganuary: Replace hyphen with dash in Bloganuary nudge text

### PR DESCRIPTION
**Pull request by wargcm**
Ref: pcdRpT-4FE-p2#comment-7965

## Description

Replaces the hyphen (-) with a dash (—) in the Bloganuary nudge text on the dashboard card. Matching Android's string change here: https://github.com/wordpress-mobile/WordPress-Android/pull/19774

## Testing

To test:
- Set device to English
- Launch Jetpack and login
- Navigate to the dashboard if necessary
- 🔎 **Verify** the Bloganuary nudge text on the dashboard card uses a dash (—) instead of a hyphen (-)

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A
PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardBloganuaryCardCell.swift
Comment (wargcm) [inline WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardBloganuaryCardCell.swift]: Given the red background, it appears GitHub thinks this is a syntax error. The file compiles fine though, but I can change this to a single line if we'd like.

Comment (wargcm) [reply WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardBloganuaryCardCell.swift]: Decided to just update it: https://github.com/wordpress-mobile/WordPress-iOS/pull/22213/commits/6155351482548b5ebd768a36b82eb281cdc81b98.

**Review by mokagio — APPROVED** [review]
Tested locally, works as expected.

IMG_85A0DC5477A4-1

@wargcm +1 for manually editing the `.strings` as the hotfix automation does not download new strings from GlotPress because GlotPress holds the source of truth for the _next upcoming version_ and as such there might be strings that were in the version an hotfix targets that are no longer in GlotPress, or new strings that are downloaded but unused.

Worth noting that:

- Until the `en.lproj` change reaches `trunk`, the GlotPress entry will still hold the version with the hyphen
- Many of the translators opted to remove the hyphen in their locale in favor of [comma](https://github.com/wordpress-mobile/wordpress-ios/blob/release/23.8.1/WordPress/Resources/es.lproj/Localizable.strings#L9571) or [semicolon](https://github.com/wordpress-mobile/wordpress-ios/blob/release/23.8.1/WordPress/Resources/it.lproj/Localizable.strings#L9568). I found this interesting. I noticed this when `git grep "Bloguanary -" -- WordPress/Resources` returned only a couple of additional entries from the `en-*` edited in the first version of this PR. I wonder if they picked up on the typo or simply felt their version better suited their locale 🤷‍♂️ 

One final note. I double checked that the dahs used here was the "em dash" not the "en dash". [The syntactic difference matters.](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use). 👌 

Method used: Copied the character from the Merriam-Webster link above, and pasted it in the search window for this diff in Chrome, verified there were 5 matches, one per each of the files touched.

image

**Comment by mokagio** [comment]
_Just waiting for green CI to before merging..._

**Pull request merged**
